src/components/squareComponent/square.spec.js

```diff
@@ -1,17 +1,90 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import GameSquare from './squareComponent';
-import Enzyme from 'enzyme';
+import Enzyme, { mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
 
 describe('Square Component', () => {
-
   it('renders the Square component without crashing', () => {
     const gameSpaceDiv = document.createElement('div');
+
     ReactDOM.render(<GameSquare />, gameSpaceDiv);
     ReactDOM.unmountComponentAtNode(gameSpaceDiv);
+
+  });
+
+  describe('Mounting the square component', () => {
+    it('it is empty when initially mounted', () => {
+      const wrapper = mount(<GameSquare />);
+
+      expect(wrapper.state('X')).toBe('');
+      expect(wrapper.state('O')).toBe('');
+
+    });
+
+  });
+
+  describe('Clicking a square', () => {
+    it('handles placing an X on the square', () => {
+      const wrapper = mount(<GameSquare />);
+
+      expect(wrapper.state('X')).toBe('');
+
+      wrapper.find('.spaces').simulate('click');
+
+      expect(wrapper.state('X')).toBe('X');
+
+    });
+
+    it('handles placing an O on the square', () => {
+      const wrapper = mount(<GameSquare />);
+
+      expect(wrapper.state('O')).toBe('');
+
+      wrapper.find('.game-spaces').simulate('click');
+
+      expect(wrapper.state('O')).toBe('O');
+
+    });
+
+  });
+
+  describe('Disabling a square', () => {
+    it('disables a square after a X is placed on the square', () => {
+      const wrapper = mount(<GameSquare />);
+      const instance = wrapper.instance();
+
+      expect(wrapper.state('disabled')).toEqual(false);
+
+      instance.toggleX();
+      wrapper.find('#space');
+
+      expect(wrapper.state('disabled')).toEqual(true);
+
+    });
+
+    it('disables a square after an O is placed on the square', () => {
+      const wrapper = mount(<GameSquare />);
+      const instance = wrapper.instance();
+
+      expect(wrapper.state('disabled')).toEqual(false);
+
+      instance.toggleO();
+      wrapper.find('#space');
+
+      expect(wrapper.state('disabled')).toEqual(true);
+
+    });
+
+  });
+
+  describe('Taking turns', () => {
```

Comment:
This test is empty. I would not expect to see any empty tests such as this committed to a repo.

Reply:
This empty test has been removed.